Second Homework SwiftUI/systempreferencesviews/SystemPreferencesView.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  SystemPreferencesView.swift
+//  Second Homework SwiftUI
+//
+//  Created by Pavlin Dimitrov on 19.11.22.
+//
+
+import SwiftUI
+
+struct SystemPreferencesView: View {
+    @ObservedObject var systemPrefrences : SystemPreferences = SystemPreferences(profile: userProfile,vpn: vpnItem,screenTime: screenTimeItem, general: generalItem, accessibility: accessibilityItem, privacyAndSecurity: privacyAndSecurityItem, passwords: passwordsItem, safari: safariItem, news: newsItem, translate: translateItem, maps: mapsItem, shortcuts: shortcutsItem)
```

Comment:
Инициализацията в дясно, може да нещо подобно:
```suggestion
    @ObservedObject var systemPrefrences : SystemPreferences = SystemPreferences.instance
```